i18n/de.toml

```diff
@@ -29,12 +29,12 @@ greenShareSub1 = "der Leistung kommt"
 greenShareSub2 = "von PV und Speicher"
 power = "Ladeleistung"
 powerSub1 = "{activeClients} von {totalClients} Nutzern"
-powerSub2 = "laden..."
+powerSub2 = "laden …"
 tabTitle = "Live-Community"
 
 [footer.savings]
-footerLong = "{percent}% Sonnenenergie"
-footerShort = "{percent}% Sonne"
+footerLong = "{percent} % Sonnenenergie"
```

Comment:
Dito bzgl. Umbruch. Warum ist das notwendig?

Reply:
Hier haben die die Regel dass wir Zahl+% immer ohne Leerzeichen schreiben. Gabs lange Diskussionen zu, aber so zieht sich das aktuell durch die gesamte Software.

Reply:
@andig @naltatis Hallo & danke!

- Laut Duden und DIN 5008 wird zwischen Zahl und Prozentzeichen stets ein Leerzeichen gesetzt: https://www.duden.de/rechtschreibung/Prozent
- Im mathematischen Formelsatz wird zwischen Zahl und Prozentzeichen – genauso wie bei Maßeinheiten – ein schmales, geschütztes Leerzeichen `U+202F` gesetzt. Leider stellen bislang nicht alle Webbrowser `U+202F` korrekt dar. Daher kommt stattdessen meist ein „normales“ oder ein „nomalbreites“, geschütztes Leerzeichen zum Einsatz. Siehe: https://de.wikipedia.org/wiki/Prozentzeichen, https://de.wikipedia.org/wiki/Hilfe:TeX#Physikalische_Gr%C3%B6%C3%9Fen

Reply:
Hier der Link zur vorangegangen Diskussion zu DIN 5008
https://github.com/evcc-io/evcc/pull/6564

Die Umstellung auf von kein Space zu einem schmalen geschützten Leerzeichen find ich prinzipiell gut. Steht hier auch auf meiner persönlichen Todo-Liste. Die komplette und konsistente Umstellung ist aber kein kleiner Aufwand. Wenn wir auf das schmale Leerzeichen umstellen dürfen keine Zahl+Prozent Kombinationen mehr in den Systemtexten sein. Wir können nicht von den Sprachpflegern erwarten, dass sie a) DIN 5008 kennen und b) wissen wie sie an das Zeichen kommen. Die Zahl+% Kombination muss dann überall in der Software erzeugt und nur als Platzhalter rausgegeben werden.

Damit wir die Diskussion nicht immer wieder führen hab ich dazu einen Issue angelegt. Der hat in meinen Augen allerdings ne sehr niedrige Priorität weil es hier kein Richtig und Falsch (trotz deutscher DIN Norm) gibt.

https://github.com/evcc-io/evcc/issues/8510

Aus diesem PR würd ich die Diskussion damit also raushalten.